test/models/user_test.rb

```diff
@@ -26,6 +26,12 @@ def setup
     refute User.new(email: test_email).valid?
   end
 
+  test 'email must be saved downcased' do
+    new_email = 'NEW_EMAIL@exaMple.cOm'
+    assert create(:user, email: new_email.downcase)
+    refute User.new(email: new_email).valid?
+  end
+
```

Comment:
@hpjaj  I tried commenting out the devise line and changing the test to:
`test 'email must be saved downcased' do
    u = build(:user, email: 'NEW_EMAIL@exaMple.cOm')
    u.save
    assert_equal 'new_email@example.com', u.email
  end`
It seemed to work and pass as a test, but I'm unsure if its needed.